roles/etcd/tasks/gen_certs_vault.yml

```diff
@@ -17,7 +17,7 @@
     issue_cert_copy_ca: "{{ item == etcd_master_certs_needed|first }}"
     issue_cert_file_group: "{{ etcd_cert_group }}"
     issue_cert_file_owner: kube
-    issue_cert_hosts: "{{ groups.etcd }}"
+    issue_cert_hosts: "{{ groups.etcd + etcd_cert_alt_names }}"
```

Comment:
I think we should move this up to line 16, 
```
 issue_cert_alt_names: "{{ groups['etcd'] + ['localhost'] + (etcd_cert_alt_names)|default() }}"
```

Reply:
Good catch